e2e/tests/components/select/default.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, test } from '../../../fixtures'
+
+test.describe('select, default', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/components/select/default')
+  })
+
+  test('initial value is selected', async ({ page }) => {
+    await expect(page.getByLabel('Sort By')).toHaveValue('updated')
```

Comment:
Hmm I guess it's case insensitive, but probably would be good to keep the case the same anyway:
```suggestion
    await expect(page.getByLabel('Sort by')).toHaveValue('updated')
```

Reply:
That should have failed :(

Reply:
Hmm seems they default to `exact: false` while Testing Library defaults to true according to the docs. Could be an oversight, I'll open an issue, doubt they'll change it now as it'd be a breaking change but maybe they can add a setting for it (assuming there isn't one already).